cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1407,6 +1410,76 @@ func maybeMountSave(disk disks.Disk, rootdir string, encrypted bool, mountOpts *
 	return true, nil
 }
 
+func generateMountsModeClassic(mst *initramfsMountsState, label string) error {
```

Comment:
this is not really generic mode classic support, as it's not measuring the model and is calling the prepare function that atm is very specific. So this should probably be renamed to something more specific.

label is very generic term, as we use it also for seed system. Is that a diskLabel?

Reply:
From initramfs-tools manpage:
 > If a label is used,  as in root=LABEL=rootPart the initrd will search all available devices for a filesystem with the appropriate label, and mount that device as the root filesystem.

From mke2fs manpage
> -L new-volume-label Set the volume label for the filesystem to new-volume-label.  The maximum length of the volume label is 16 bytes.

It's not quite `diskLabel`, nor `gptLabel`, but it is rather specifically `fsLabel`, or maybe even `rootFsLabel`.

Reply:
> this is not really generic mode classic support

Given all the other architectural plans, i do agree that this is no longer generic hybrid classic mode. But rather something specific to basic, optionally encrypted, classic cloud image.

What should I call it? `generateMountsModeCloudCVM()` would that be suitable?

In that case I can also drop label argument and make it a constant, that is specific to ModeCloudCVM.

Reply:
yes, something like generateMountsModeRunCloudCVM seems appropriate